src/components/BlogPostsSection.js

```diff
@@ -2,55 +2,58 @@ import * as React from "react"
 import { Link } from "gatsby"
 
 const BlogPostsSection = ({ type, text }) => {
+  let renderSection;
```

Comment:
What do you think about this change?
```
const sections = {
    "blog": (
      <div className="section-title text-center">
        <h2>All Blog Posts</h2>
        <ul className="breadcrumb-nav">
          <li><Link to="/">Home</Link></li>
          <li>All blog posts</li>
        </ul>
      </div>
    ),
    "category": (
      <div className="section-title text-center">
        <h2>Blog Posts by Category</h2>
        <ul className="breadcrumb-nav">
          <li><Link to="/">Home</Link></li>
          <li><Link to="/blog">Blog</Link></li>
          <li>Category: {text}</li>
        </ul>
      </div>
    ),
    "tag": (
      <div className="section-title text-center">
        <h2>Blog Posts by Tag</h2>
        <ul className="breadcrumb-nav">
          <li><Link to="/">Home</Link></li>
          <li><Link to="/blog">Blog</Link></li>
          <li>Tag: {text}</li>
        </ul>
      </div>
    ),
    "search": (
      <div className="section-title text-center">
        <h2>Search Results</h2>
        <ul className="breadcrumb-nav">
          <li><Link to="/">Home</Link></li>
          <li><Link to="/blog">Blog</Link></li>
          <li>Search: {text}</li>
        </ul>
      </div>
    )
  }

  const renderSection = sections[type]

  if (!renderSection) {
    throw new Error("Invalid type for blog post section");
  }
```

Reply:
There will be the same result as my code.

Reply:
Yes, but I think it would be easier to read